### PR TITLE
1.0 fix video block

### PR DESCRIPTION
**Pull request by itkg-ngilain**

### CODE REVIEW

**Comment by SOLenG** [comment]
:+1:

File: Backoffice/GenerateForm/Strategies/VideoStrategy.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
  */
 class VideoStrategy extends AbstractBlockStrategy
 {
-    const TEMPLATE = 'OpenOrchestraBackofficeBundle:Block/Video:form.html.twig';
+    const TEMPLATE = 'OpenOrchestraBackofficeBundle:Block:Video/Form/form.html.twig';
```
Comment (nicolasThal) [inline Backoffice/GenerateForm/Strategies/VideoStrategy.php]: I would rather put Bundle:Folders:file ?

Comment (itkg-ngilain) [inline Backoffice/GenerateForm/Strategies/VideoStrategy.php]: Both are used on the project but I think it's a good idea to have the same logic everywhere. I'm ok with your proposal.

**Comment by itkg-ngilain** [comment]
@nicolasThal : updated

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**